src/cbc_sdk/platform/alerts.py

```diff
@@ -1638,6 +1640,13 @@ def _build_request(self, from_row, max_rows, add_sort=True):
         """
         request = super(GroupedAlertSearchQuery, self)._build_request(from_row, max_rows, add_sort=True)
         request["group_by"] = {"field": self._group_by}
+
+        # build GroupedAlert facet request
+        if self._fieldlist is not []:
```

Comment:
Don't add this to _build_request as it breaks the query object as soon as you call facets. If a developer called facet then wanted to query the objects it would break the query search. If you are going to follow the AlertSearchQuery way then move this inside the facet function